### PR TITLE
📝 Update includes for `docs/en/docs/tutorial/extra-data-types.md`

### PR DESCRIPTION
**Pull request by celestywang**
📝 Update includes in `docs/en/docs/tutorial/extra-data-types.md`

### CODE REVIEW

**Comment by celestywang** [comment]
@alejsdev Could you please approve the workflow for this PR? Thank you!

File: docs/en/docs/tutorial/extra-data-types.md [hunk 1]
```diff
@@ -55,108 +55,4 @@ Here are some of the additional data types you can use:
...
-
-///
-
-```Python hl_lines="1  2  12-16"
-{!> ../../docs_src/extra_data_types/tutorial001.py!}
-```
-
-////
-
-Note that the parameters inside the function have their natural data type, and you can, for example, perform normal date manipulations, like:
```
Comment (tiangolo) [inline docs/en/docs/tutorial/extra-data-types.md:110]: This text should be kept, the include below is a different one from the previous one.

**Review by tiangolo — CHANGES_REQUESTED** [review]

File: docs/en/docs/tutorial/extra-data-types.md (hunk 1 above)
Comment (celestywang) [reply docs/en/docs/tutorial/extra-data-types.md:110]: @tiangolo thank you for the feedback. I have updated it!

**Comment by tiangolo** [comment]
Thanks for the effort! :cake: 

This was handled in another PR, so I'll close this one now. :coffee:

**Pull request closed without merging**